SEImplementation/python/sourcextractor/config/measurement_config.py

```diff
@@ -82,37 +82,49 @@ def load_fits_image(self, image, psf=None, weight=None, **kwargs):
         :return: A ImageGroup representing the images
         """
 
-        image_hdu_list = FitsFile(image)
-        image_hdu_idx = image_hdu_list.hdu_list
+        image_file = FitsFile(image)
+        if "image_hdu" in kwargs.keys():
+            image_hdu_list = [kwargs["image_hdu"]]
+            del kwargs["image_hdu"]
```

Comment:
You can combine these with [`kwargs.pop('image_hdu')`](https://docs.python.org/3/library/stdtypes.html#dict.pop)